mrbgem.rake

```diff
@@ -49,13 +49,13 @@ MRuby::Gem::Specification.new('mruby-docopt') do |spec|
           run_command e, "sed -i -e 's/\\/usr\\/bin\\//\\/opt\\/osxcross\\/target\\/bin\\/x86_64-apple-darwin14-/' CMakeFiles/docopt_s.dir/link.txt"
         end
       end
-      run_command e, "make"
+      run_command e, "make docopt_s"
     end
   end
 
   spec.cxx.include_paths << docopt_dir
   spec.cxx.include_paths << File.join(File.dirname(__FILE__), "include")
   spec.cxx.flags << "-std=c++11"
   spec.build.linker.library_paths << docopt_dir
-  spec.build.linker.flags_after_libraries << "-ldocopt_s"
+  spec.build.linker.flags_after_libraries << "-ldocopt"
```

Comment:
I thinking making it explicit is better. Linking to dynamic would lead to hard to debug errors down the line since you'll be using the binaries out of the docker image.